extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -244,7 +245,7 @@ export class AmpStoryPage extends AMP.BaseElement {
     );
 
     /** @private {?boolean}  */
-    this.isFirstPage_ = null;
+    this.isActivePage_ = null;
```

Comment:
```suggestion
    this.isPrerenderActivePage_ = null;
```

active page has a different meaning

Reply:
Changed this on both files 👍